app/src/main/java/org/mozilla/fenix/home/recenttabs/view/RecentTabs.kt

```diff
@@ -213,8 +214,9 @@ fun RecentTabImage(
         }
         else -> ThumbnailCard(
             url = tab.state.content.url,
-            key = tab.state.id,
-            modifier = modifier
+            key = tab.state.content.url.hashCode().toString(),
```

Comment:
This should actually be using `tab.state.id`

Reply:
As discussed, your correct that this key should not be changed.  I'll remove this part and open a new issue for the content scale fix.  Thanks